### DISCUSSION TITLE
bad performance in windows 11

### DISCUSSION THREAD

**Discussion by AlBundy33** [utmapp/UTM#6589] [Windows]
I've installed UTM on my Intel Macbook Pro (latest MacOS) und tried to run a Windows VM to compare it against Parallels Desktop.

In Parallels Desktop the performance is like on my bare metal windows pc.
In UTM all is horribly slow (openen start menu, starting explorer even moving windows is slow).

I've created a new VM in UTM, selected virtualization and that I wan't to install Windows 10 or above (for the rest I've used the defaults).

Is the something I can change to improve performance?
I've tried some different CPUs and Graphic settings, set 12GB RAM and 6 cores and installed the guest tools but none of them helped.

To be honest for my first test I've converted my parallels disk with 
```
qemu-image convert -f parallels pathtodiskimage.hds -O qcpw2 test.qcow2
```

Then I've tried to create a new vm an install windows from scratch but even the installer seems to be slow.

since windows runs as expected in parallels I'm pretty sure that this works also with qemu - I "only" need the correct settings. 🙁 

I hope someone has a solution for this problem.

**Comment by AlBundy33** [comment]
maybe I can check #5468
-> checked but didn't help

**Comment by osy** [comment]
This is expected because we currently do not support GPU acceleration in Windows.
